app/src/main/java/com/yacgroup/yacguide/TableActivity.kt

```diff
@@ -37,6 +37,8 @@ abstract class TableActivity : BaseNavigationActivity() {
 
     open fun showComments(v: View) {}
 
+    open fun showParent(v: View) {}
```

Comment:
That would remove the activity from the stack and prevent navigating back via normal controls. (e.g. navigating from the tourbook to an ascend and back)

Reply:
Yes, but why should you go back to a child instead of selecting it again? Secondly, I could imagine that this "history stack" might be dangerous. Imagine this: child activity -> switch to parent -> select child regularly again -> change child's content -> go back to parent via back button -> go back to old version of child via back button. Now the state of the old child might be inconsistent since we have changed e.g. some data that the old child version now tries to access.

Reply:
For that case, I think we should make sure that the onResume / onPause events properly handle the foreground / background switching and redraw the activity.